programs/bpf_loader/src/bpf_verifier.rs

```diff
@@ -58,10 +58,6 @@ fn check_prog_len(prog: &[u8]) -> Result<(), BPFError> {
     if prog.len() % ebpf::INSN_SIZE != 0 {
         return Err(VerifierError::ProgramLengthNotMultiple.into());
     }
-    if prog.len() > ebpf::PROG_MAX_SIZE {
```

Comment:
This has a risk of diverging behaviour right?   A new BPF program could be accepted by less than 66.6% of the nodes during an upgrade.

Reply:
Yeah I think this needs to be feature gated :-/

Reply:
Hmm, yeah, I was thinking about it as there would be no programs on the chain larger then this that would also be able to execute successfully with the current budget, but yeah, I guess finalize could diverge.  Divergence avoidance is hard